client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -16,12 +17,14 @@ import Masterbar from 'layout/masterbar/masterbar';
 import './style.scss';
 
 export default function() {
+	const translate = useTranslate();
+
 	return (
 		<Masterbar>
 			<Item
 				className="masterbar__item-home"
 				url="/"
-				tooltip="Jetpack Cloud Homepage" /* @todo: localize the string */
+				tooltip={ translate( 'Jetpack Cloud Homepage' ) }
```

Comment:
Are we sure we want to expose the "homepage" term to users? I'd assume something like "dashboard" would make more sense - we use it for the nav item anyway. Or it could be removed completely, leaving it just "Jetpack Cloud". What do you think?

Reply:
Totally 👍 I've replaced it with a "Dashboard" in c1c158c since this is a term we're using in the sidebar, too.